diff --git a/data/npclib/npc_system/bank_system.lua b/data/npclib/npc_system/bank_system.lua
--- a/data/npclib/npc_system/bank_system.lua
+++ b/data/npclib/npc_system/bank_system.lua
@@ -160,9 +160,7 @@ function Npc:parseBank(message, npc, creature, npcHandler)
 				npcHandler:say(string.format("Are you sure you wish to withdraw %d gold from your bank account?", count[playerId]), npc, creature)
 				npcHandler:setTopic(playerId, 7)
 			else
-				local messageText = count[playerId] and count[playerId] > maximumWithdrawal
-					and string.format("You can withdraw at most %d gold at a time.", maximumWithdrawal)
-					or "There is not enough gold on your account."
+				local messageText = count[playerId] and count[playerId] > maximumWithdrawal and string.format("You can withdraw at most %d gold at a time.", maximumWithdrawal) or "There is not enough gold on your account."
 				npcHandler:say(messageText, npc, creature)
 				npcHandler:setTopic(playerId, 0)
 			end
@@ -178,9 +176,7 @@ function Npc:parseBank(message, npc, creature, npcHandler)
 			npcHandler:say(string.format("Are you sure you wish to withdraw %d gold from your bank account?", count[playerId]), npc, creature)
 			npcHandler:setTopic(playerId, 7)
 		else
-			local messageText = count[playerId] and count[playerId] > maximumWithdrawal
-				and string.format("You can withdraw at most %d gold at a time.", maximumWithdrawal)
-				or "There is not enough gold on your account."
+			local messageText = count[playerId] and count[playerId] > maximumWithdrawal and string.format("You can withdraw at most %d gold at a time.", maximumWithdrawal) or "There is not enough gold on your account."
 			npcHandler:say(messageText, npc, creature)
 			npcHandler:setTopic(playerId, 0)
 		end
